extensions/avro/deployment/src/main/java/io/quarkus/avro/deployment/AvroSchemaCodeGenProvider.java

```diff
@@ -47,7 +48,7 @@ void compileSingleFile(Path filePath,
         // allow them to share a single schema so reuse and sharing of schema
         // is possible.
         try {
-            if (options.imports == EMPTY) {
+            if (ArrayUtils.isEmpty(options.imports)) {
```

Comment:
We try to avoid depending on commons things in Quarkus (even if it's in the classpath because of other dependencies forcing it on us). Could you rewrite that with plain Java?